src/lib/ibcclient.ts

```diff
@@ -361,6 +364,17 @@ export class IbcClient {
     return this.tm.commit(height);
   }
 
+  /// Returns the unbonding period in seconds
+  public async getUnbondingPeriod(): Promise<number> {
+    const params = await this.query.staking.params();
+    const seconds = params?.unbondingTime?.seconds?.toNumber();
+    if (!seconds) {
```

Comment:
This also throws for 0, is that intended?

Reply:
Yes, 0 unbonding period is either impossible of incompatible with IBC. (You need several minutes at absolute minimum)